unlock-js/src/web3Service.js

```diff
@@ -77,13 +77,15 @@ export default class Web3Service extends UnlockService {
     }
 
     this.inputsHandlers = {
-      createLock: async (transactionHash, contractAddress, params) => {
+      createLock: async (transactionHash, contractAddress, sender, params) => {
         // The annoying part here is that we do not have the lock address...
         // Since it is not an argument to the function.
         // We will 'guess' it again using generateLockAddress
         // knowing that this may create a race condition another lock creation pending transaction
```

Comment:
With the change to `create2` it's no longer a "guess".  If there's a conflict (e.g. same user reusing a salt) the createLock transaction will fail.

Reply:
Good point! fixed the comment,